backend/src/main/kotlin/app/ehrenamtskarte/backend/graphql/application/EakApplicationMutationController.kt

```diff
@@ -235,7 +234,7 @@ class EakApplicationMutationController(
     ): Boolean =
         transaction {
             val application =
-                ApplicationEntity.findById(applicationId) ?: throw NotFoundException("Application not found")
```

Comment:
removed NotFoundException from queries/mutations because it is mapped to "seite nicht gefunded" error in administration